test/BalanceTx/Time.purs

```diff
@@ -132,24 +130,18 @@ validityToPosixTime { validityStartInterval, ttl: timeToLive } =
         case timeToLive of
           Just end ->
             let
-              end' = substractOne end
+              end' = subtractOne end
```

Comment:
Is there a guarantee that end is > 1? Why? each unsafePartial call should come with a comment, if it's not obvious.